app/models/planet.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy.orm import Mapped, mapped_column
+from ..db import db
+
+class Planet(db.Model):
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    name: Mapped[str]
+    description: Mapped[str]
+    num_moons: Mapped[int]
```

Comment:
We could have some methods on this class that could make our business logic in our routes more clean and in line with SRP